ios/Classes/SwiftDeviceCalendarPlugin.swift

```diff
@@ -23,6 +23,16 @@ extension String {
     }
 }
 
+fileprivate class SingleEventStore {
+    public let eventStore: EKEventStore
+    
+    public static let shared = SingleEventStore()
+
+    private init() {
+        eventStore = EKEventStore()
+    }
```

Comment:
Doesn't this initialise two instances of EKEventStore? 

1: `eventStore` which doesn't appear to be used, and 2: `shared`

Reply:
I don't think so. The whole point of the Singleton pattern is that a class is instantiated only once. In my code the only point of access on `SingleEventStore` class is through its static instance `shared`, which the private constructor that instantiates an `eventStore`.